### ISSUE TITLE
Remove the "system bridge" text from linux tray icon

### ISSUE THREAD

**Issue by Sipulikorva** [timmo001/system-bridge#3341]
### Description

Could you consider removing the "system bridge" text from the tray icon, just leave the icon - or at least have the option to toggle the text. It does look awful like this.
[image](https://github.com/user-attachments/assets/0ae3316b-1a33-433d-bd8e-cf711d86d4db)

**Comment by Sipulikorva** [comment]
I don't think it is, since this is the only program I've seen that has this sort of title on tray. 

I'm taking a guess here since I mostly don't understand the code, but could it be the title here that makes the text appear:

https://github.com/timmo001/system-bridge/blob/9acd5d3c7d000254830edde96ff425d1ce8ab37b/src-tauri/tauri.conf.json#L10-L17

**Comment by ooTruffle** [comment]
i say de as on kde plasma it only shows the icon
![image](https://github.com/user-attachments/assets/f36074e0-caab-47b0-ac52-62b3cedbf2d5)
